apps/fyle/views.py

```diff
@@ -254,3 +254,29 @@ def post(self, request, *args, **kwargs):
                 },
                 status=status.HTTP_400_BAD_REQUEST
             )
+
+
+class ExpenseFilterView(generics.ListCreateAPIView):
+    """
+    Expense Filter view
+    """
+    serializer_class = ExpenseFilterSerializer
+
+    def get(self, request, *args, **kwargs):
```

Comment:
This isn't correct, if we define the `get` function then the whole point of using generic view goes away
```suggestion
    class ExpenseFilterView(generics.ListCreateAPIView):
    """
    Expense Filter view
    """
    serializer_class = ExpenseFilterSerializer
    
    def get_queryset(self):
        queryset = ExpenseFilter.objects.filter(workspace_id=self.kwargs['workspace_id'])
        return queryset
```

That's all, we don't need the whole function you've written